how-to/how-to-use-rootless-kata-containers-with-podman.md

```diff
@@ -0,0 +1,171 @@
+# How to use rootless Kata Containers with Podman
+
+* [How to use rootless Kata Containers with Podman](#how-to-use-rootless-kata-containers-with-podman)
+    * [Requirements](#requirements)
+    * [Installation](#installation)
+    * [Configuration](#configuration)
+            * [1. Disable SELinux](#1-disable-selinux)
+            * [2. Add user to KVM group](#2-add-user-to-kvm-group)
+            * [3. Reboot](#3-reboot)
+            * [5. Disable `vhost-net`](#5-disable-vhost-net)
+            * [6. Modify the Kata images permissions](#6-modify-the-kata-images-permissions)
+            * [7. Set up Podman rootless configuration](#7-set-up-podman-rootless-configuration)
+            * [8. Add Kata Runtime to Podman configuration file (optional)](#8-add-kata-runtime-to-podman-configuration-file-optional)
+            * [9. Set Kata runtime as Podman's default OCI runtime (optional)](#9-set-kata-runtime-as-podmans-default-oci-runtime-optional)
+    * [Run Kata with rootless Podman](#run-kata-with-rootless-podman)
+    * [Appendix: Possible Errors](#appendix-possible-errors)
+        * [Error caused by agent or runtime version mismatch](#error-caused-by-agent-or-runtime-version-mismatch)
+        * [Missing registry file](#missing-registry-file)
+
+
+For an even more secure system, [Kata Containers](https://Katacontainers.io) can run workloads without a privileged user. Using [Podman](https://podman.io/) as the container engine, and [slirp4netns](https://github.com/rootless-containers/slirp4netns) for user-space networking.
+
+## Requirements
+- A Linux system, see [supported distributions](https://github.com/kata-containers/documentation/blob/master/install/README.md#supported-distributions) for an updated list.
+  - If using CentOS 7, `newuidmap` and `newgidmap` do not exist, and can be installed with:
```

Comment:
Suggested rewrite: 
If using CentOS 7, `newuidmap` and `newgidmap` do not exist. Install them with: